ignite-base/App/Themes/Colors.js

```diff
@@ -22,4 +24,8 @@ const colors = {
   drawer: 'rgba(30, 30, 29, 0.95)'
 }
 
+const colors = merge(igniteColors, {
+  // add colors for your Ignited app here
+})
+
```

Comment:
No, not too petty at all. This furthers my intent of decoupling all ignite flavors from the ignite-generated app. I want users to be able to open ignite screens such as the presentationScreen and access all of the useful screens, such as componentExamples, while being able to set up the flavors for their own app, with minimal conflicts.